css/email-signup-mobile.css

```
@@ -0,0 +1,269 @@
+@media (max-width: 768px) {
+  /* Ensure proper keyboard behavior */
+  .notification-form {
+    /* Prevent form zoom on older iOS */
+    transform: translateZ(0);
+    -webkit-transform: translateZ(0);
+  }
+
+  /* Input attributes for optimal keyboard */
+  .notification-form input[type="email"] {
+    /* Disable autocorrect and autocapitalize for email */
+    autocorrect: off;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)